src/components/dialogs/parameters/security-analysis/security-analysis-parameters.tsx

```diff
@@ -123,12 +139,19 @@ export const SecurityAnalysisParameters: FunctionComponent<{
         [params]
     );
 
-    const { handleSubmit, formState } = formMethods;
+    const { handleSubmit, formState, reset } = formMethods;
 
     const updateLimitReductions = useCallback(
         (formLimits: Record<string, any>) => {
             updateParameters({
                 ...params,
+                [PARAM_SA_FLOW_PROPORTIONAL_THRESHOLD]: formLimits[PARAM_SA_FLOW_PROPORTIONAL_THRESHOLD] / 100,
+                [PARAM_SA_LOW_VOLTAGE_PROPORTIONAL_THRESHOLD]:
+                    formLimits[PARAM_SA_LOW_VOLTAGE_PROPORTIONAL_THRESHOLD] / 100,
+                [PARAM_SA_LOW_VOLTAGE_ABSOLUTE_THRESHOLD]: formLimits[PARAM_SA_LOW_VOLTAGE_ABSOLUTE_THRESHOLD],
+                [PARAM_SA_HIGH_VOLTAGE_PROPORTIONAL_THRESHOLD]:
+                    formLimits[PARAM_SA_HIGH_VOLTAGE_PROPORTIONAL_THRESHOLD] / 100,
+                [PARAM_SA_HIGH_VOLTAGE_ABSOLUTE_THRESHOLD]: formLimits[PARAM_SA_HIGH_VOLTAGE_ABSOLUTE_THRESHOLD],
```

Comment:
strange to update general parameters in `updateLimitReductions` function ?

Reply:
i renamed the function to 'updateSaParametters'